fixtures/tracks/animal/SETUP.md

```diff
@@ -0,0 +1 @@
+This is the content of the setup.md file.
```

Comment:
Oh right, this file is never displayed because it is overridden by the track_hints file.
Maybe you could add that info to the setup.md file as well?
It certainly confused me.